src/quafu/results/results.py

```diff
@@ -11,47 +11,46 @@ class Result(object):
 
 
 class ExecResult(Result):
-    """
+    """ 
     Class that save the execute results returned from backend.
-
+    
     Attributes:
         counts (dict): Samples counts on each bitstring.
         probabilities (dict): Calculated probabilities on each bitstring.
         taskid (int): Unique task id for the execute result.
         transpiled_circuit (QuantumCircuit): Quantum circuit transpiled on backend.
     """
 
-    def __init__(self, input_dict, measures):
+    def __init__(self, input_dict):
         status_map = {0: "In Queue", 1: "Running", 2: "Completed", "Canceled": 3, 4: "Failed"}
-        self.measures = measures
-        self.task_status = status_map[input_dict["status"]]
-        self.res = eval(input_dict['res'])
-        self.counts = OrderedDict(sorted(self.res.items(), key=lambda s: s[0]))
-        self.logicalq_res = {}
-        cbits = list(self.measures.values())
-        indexed_cbits = {bit: i for i, bit in enumerate(sorted(cbits))}
```

Comment:
The `indexed_cbits` and `squeezed_cbits` are deleted, I suggest to keep this tricky implementation to avoid index error.

details: https://github.com/ScQ-Cloud/pyquafu/pull/51

Reply:
Yes, that's overwritten by mistake... I am modyfing it back..